CoffeeChat/Controller/HomeViewController.swift

```diff
@@ -82,114 +40,98 @@ class HomeViewController: UIViewController {
         }
 
-        titleLabel.snp.makeConstraints { make in
-            make.centerX.equalToSuperview()
-            make.top.equalTo(view.safeAreaLayoutGuide).offset(50)
+        tabContainerView.snp.makeConstraints { make in
+            make.leading.trailing.bottom.equalToSuperview()
+            make.top.equalTo(tabCollectionView.snp.bottom)
         }
+
     }
 
-    @objc private func profilePressed() {
-        let menu = SideMenuNavigationController(rootViewController: ProfileMenuViewController())
-        let presentationStyle: SideMenuPresentationStyle = .viewSlideOutMenuPartialIn
-        presentationStyle.presentingEndAlpha = 0.85
-        menu.presentationStyle = presentationStyle
-        menu.leftSide = true
-        menu.statusBarEndAlpha = 0
-        menu.menuWidth = view.frame.width * 0.8
-        present(menu, animated: true, completion: nil)
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        navigationController?.setNavigationBarHidden(true, animated: true)
     }
 
-    @objc private func logoutPressed() {
-        GIDSignIn.sharedInstance().signOut()
+}
+
+extension HomeViewController: UICollectionViewDelegate {
 
-        let loginVC = LoginViewController()
-        loginVC.modalPresentationStyle = .fullScreen
-        present(loginVC, animated: true, completion: nil)
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        activeTabIndex = indexPath.item
+        tabPageViewController.setViewController(to: indexPath.item)
+        tabCollectionView.reloadData()
     }
 
-    @objc private func reachOutPressed() {
-        let timeVC = SchedulingTimeViewController(isConfirming: true, isPicking: false)
-        navigationController?.pushViewController(timeVC, animated: true)
-    }
 }
 
-extension HomeViewController: UITableViewDataSource {
-    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        matchSummaries.count
+extension HomeViewController: UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return tabs.count
     }
 
-    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        guard let cell = tableView.dequeueReusableCell(withIdentifier: cellReuseId, for: indexPath) as? MatchSummaryTableViewCell else { return UITableViewCell() }
-        let summary = matchSummaries[indexPath.row]
-        cell.configure(for: summary)
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: tabCellReuseIdentifier, for: indexPath) as? HomeTabOptionCollectionViewCell else { return UICollectionViewCell() }
+        if indexPath.item == activeTabIndex {
+            cell.isSelected = true
+        }
```

Comment:
`cell.isSelected = indexPath.item == activeTabIndex` unless you want it to stay true even after `indexPath.item == activeTabIndex` is no longer true.

Reply:
Oh ya so, I its just cause I made the other screen a diff color to show the page scroll. The actual page will have the same color and will have more space at the top so it shouldn't look so abrupt.